core/src/services/gdrive/core.rs

```diff
@@ -396,6 +396,9 @@ impl PathQuery for GdrivePathQuery {
     async fn create_dir(&self, parent_id: &str, name: &str) -> Result<String> {
         let url = "https://www.googleapis.com/drive/v3/files";
 
+        // trim "/" at the end of name
+        let name = name.trim_end_matches("/");
```

Comment:
Only trim name for `create_dir` is not enough, we could push `name` with `/` in cache at:

https://github.com/apache/opendal/blob/c82dc139e7478ac96a90c49dc8e5326ecc057d75/core/src/raw/path_cache.rs#L183-L192

Reply:
good call. will update

Reply:
Updated.

Not sure whether it's on the right direction.

Feel free to correct me if I were wrong.

Thanks a lot!